Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/README.md

```diff
@@ -1127,7 +1127,7 @@ For more details about the authentication used in this integration, see <a href=
 </div>
 <div class="cl-preview-section"><hr></div>
 <div class="cl-preview-section">
-<p>Changes the user password. Permissions: Directory.AccessAsUser.All(Delegated)</p>
+<p>Changes the user password. Can only work with a self-deployed application and the permission: Directory.AccessAsUser.All(Delegated)</p>
```

Comment:
i believe this is true also for `msgraph-user-terminate-session`, please make sure you add it to all related commands, and also need to add it to the commands description